AsBuiltReport.Fortinet.FortiGate.Style.ps1

```diff
@@ -49,11 +49,11 @@ TableStyle -Id 'Borderless' -HeaderStyle Normal -RowStyle Normal -BorderWidth 0
 # Header & Footer
 if ($ReportConfig.Report.ShowHeaderFooter) {
     Header -Default {
-        Paragraph -Style Header "$($ReportConfig.Report.Name) - v$($ReportConfig.Report.Version)"
```

Comment:
@tpcarman this change is ok ?

Reply:
The report style script is now redundant as it it replaced by the default style script in AsBuiltReport.Core v1.4.0 module, so users would not see this change unless they specified this style script.